### PR TITLE
remove invalid samples products

### PR DESCRIPTION
**Pull request by daxianji007**

### CODE REVIEW

**Review by RezaJooyandeh — APPROVED** [review]
I am guessing this is to unblock sample publishing?

**Review by heaths — CHANGES_REQUESTED** [review]
Removing these won't achieve the desired effect.

@christothes who can we tag about metrics-advisor?

Someone is already on the hook to get a product ID for communication.

File: sdk/metricsadvisor/Azure.AI.MetricsAdvisor/samples/README.md [hunk 1]
```diff
@@ -5,7 +5,6 @@ languages:
 products:
 - azure
 - azure-cognitive-services
-- azure-metrics-advisor
```
Comment (kinelski) [inline sdk/metricsadvisor/Azure.AI.MetricsAdvisor/samples/README.md:8]: I'm not following. Why is removing this needed?

**Review by kinelski — CHANGES_REQUESTED** [review]

File: sdk/metricsadvisor/Azure.AI.MetricsAdvisor/samples/README.md (hunk 1 above)
Comment (kinelski) [reply sdk/metricsadvisor/Azure.AI.MetricsAdvisor/samples/README.md:8]: Let's go with `azure-cognitive-services` for now. I can update it again once we have the `azure-metrics-advisor` ID.

**Review by kinelski — APPROVED** [review]
Metrics Advisor change LGTM.

**Comment by jsquire** [comment]
Hi @daxianji007.  There hasn't been recent engagement on this PR.  Would you please be so kind as to let us know if this is still an active work stream or if the PR should be closed out?

**Comment by jsquire** [comment]
Since there hasn't been recent engagement, I'm going to close this out.  Please feel free to reopen if you'd like to continue working on these changes.

**Pull request closed without merging**